tests/testFocalPlane.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+#
+# LSST Data Management System
+# Copyright 2008-2013 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+import numpy
+import unittest
+
+import lsst.daf.base as dafBase
+import lsst.afw.image as afwImage
+from lsst.afw.cameraGeom.testUtils import DetectorWrapper
+import lsst.utils.tests
+import lsst.meas.base.tests
+from lsst.meas.base.tests import (AlgorithmTestCase, SingleFramePluginTransformSetupHelper)
+
+class FocalPlaneTestCase(AlgorithmTestCase):
```

Comment:
This test is basically fine, but it might be even better if you were to spread a few randomly positioned sources over the image and demonstrate you got back the correct focal plane coordinates for each one rather than just this hard-coded special case.